src/video.cpp

```diff
@@ -37,7 +37,7 @@ CVideo* CVideo::singleton_ = nullptr;
 
 namespace
 {
-surface frameBuffer = nullptr;
+surface frameBuffer = surface();
```

Comment:
```
surface frameBuffer;
```

Reply:
I did this a lot. I'll run through and check them all.